libcodechecker/server/run_db_model.py

```diff
@@ -198,6 +217,9 @@ class Report(Base):
                                    'reopened',
                                    name='detection_status'))
 
+    detected_at = Column(DateTime)
```

Comment:
I believe a `NOT NULL` constraint should exist on the `detected_at` column? Every report is expected to have at least a detection date.